### DISCUSSION TITLE
What country does "The Hunger Games" take place in?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#231] [Galaxy Trivia] [answered]
**Hint:** 🍞🏆

**Comment by KelechiOdom10** [comment, marked as answer]
Panem perhaps

**Reply by ParanoidUser** [reply]
Bigno! Panem derives from the Latin phrase *Panem et circenses*, which literally translates into 'bread and circuses'. One more to go for the Galaxy Brain badge. 😊
